modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -1205,15 +1205,20 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
   };
 
   const isChildOf = (node: Node, parent: Node) => {
-    while (node) {
-      if (parent === node) {
-        return true;
+    // IE only supports the native `contains` functions on Elements, not Nodes
+    if (!isIE || NodeType.isElement(parent)) {
```

Comment:
This just makes sure we use the native `Node.contains` function if it's available/viable as it's ~5x faster on Chrome.

Note: I made this change to account for the fact I switched to using `isChildOf` in `RangeWalk` which used to cap how far it'd walk, so using `isChildOf` would have had worse performance (it's also just a good modernisation change).

Reply:
Are we also able to short-circuit in the case where parent isn't an `Element`? If not, still a good win

Reply:
Not sure what you mean there sorry? So on every browser except IE 11 it'll use the `Node.contains` path and for IE 11 it'll use `Element.contains` so it'd only do the slower walking approach if the passed in parent is not an `Element`.

Reply:
Turned out this is why my tests were failing... IE 11 doesn't handle `node` being a text node, or similar. As such I've just changed it to always use the slow path on IE since it also turns out it doesn't work with non HTML elements (eg SVGs).